scsi_defs.h

```diff
@@ -18,6 +18,7 @@
 #define READ_DISC_INFORMATION           0x51
 #define MODE_SELECT_10                  0x55
 #define MODE_SENSE_10                   0x5a
+#define RECEIVE_COPY_RESULTS            0x84
```

Comment:
I just see that in SPC5, this has been renamed to RECEIVE COPY STATUS.
BTW, which one should we use will be better ?